app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink } from 'redux-first-router-link';
```

Comment:
Link

Reply:
Is it a actually `NavLink` here, isn't it?